R/utm-convert.R

```diff
@@ -0,0 +1,103 @@
+#' Convert a data.frame of UTM coordinates to an sf object with a single CRS
+#'
+#' This can operate on a data frame containing coordinates from multiple UTM zones
+#' with a column denoting the zone, or a single zone for the full dataset.
+#'
+#' It supports data collected in either the NAD83 or WGS84 ellipsoid in
+#' the Northern hemisphere
+#'
+#' @param x data.frame containing UTM coordinates, with a zone column
+#' @param xcol the name of the 'easting' columnm
+#' @param ycol the name of the 'northing' column
```

Comment:
Any reason why we aren't just calling these `easting_col` and `northing_col`? Seems more direct for a user. 
```suggestion
#' @param xcol the name of the 'easting' column
#' @param ycol the name of the 'northing' column
```

Reply:
Yeah that's probably better. The other option would be to do it the same as `sf` and have one `coords` argument to which you pass a two-element character vector of the column names?

Reply:
That could totally work. I always have to pause and remember the order of that vector when I use that function so I'd lean towards separate arguments.

Reply:
Yeah I agree

Reply:
I just went with `"easting"` and `"northing"` for the column argument names